packages/shared/src/modules/contributors/libs/types/contributor-patch-request-dto.type.ts

```diff
@@ -1,4 +1,5 @@
 type ContributorPatchRequestDto = {
+	hiddenAt: null | string;
```

Comment:
for request it's better to pass isHidden, and set the timestamp on the backend side 
```suggestion
	isHidden: boolean;
```